src/main/kotlin/no/nav/familie/ef/sak/vedtak/uttrekk/UttrekkArbeidssøkerDto.kt

```diff
@@ -0,0 +1,27 @@
+package no.nav.familie.ef.sak.vedtak.uttrekk
+
+import java.time.LocalDateTime
+import java.time.YearMonth
+import java.util.UUID
+
+data class UttrekkArbeidssøkereDto(
+        val årMåned: YearMonth,
+        val antallTotalt: Int,
+        val antallKontrollert: Int,
+        val arbeidssøkere: List<UttrekkArbeidsssøkerDto>
+)
```

Comment:
Jeg tror det kommer finnes ett behov for paginering når flere kommer på ny løsning. Jeg har lagt til støtte for det i responsen, men har ikke lagt til funksjonalitet i api/mot databasen i denne runde.